contents/blog/how-come-we-ship-so-much.md

```diff
@@ -0,0 +1,129 @@
+
+## 7. We are a late mover
+
+I've had lots of founders tell me that they can't pursue idea X because some other startups are tackling it. 
+
+This makes me laugh because we simply won't build a new product unless there is a $1bn+ competitor already offering it. We _love_ competition. It shows you there's demand. 
+
+The magic in our platform (after an individual product is working well for our users) is then integrating everything together.
+
+For non-God tier product people, such as me, I think it's easier to build a better version of X than it is to create something entirely new.
+
+## 8. We raised venture capital
+
+Obviously, this matters. We didn't raise or spend a ridiculous amount compared to other companies we compete with – we have done a $3M seed, $9M series A and a $15M series B. We have a lot of it left and will be profitable next year.
+
+Raising money does let you build more stuff, but whether you _should_ is very questionable and [a topic for another time](/blog/vc-or-bootstrap).
+
+## 9. Trust and feedback over process
+
+This is one of [our values](/handbook/company/values). It's simply up to the person building in most situations. Building and scaling something people want is a nuanced problem, so we let people use their judgement. When they get it wrong, we are direct and give feedback.
+
+To quote one of our team: "process is scar tissue" – it often overcorrects, but because [humans are risk adverse](https://www.adamjuliangoldstein.com/blog/anxiety-algorithm/) it feels natural to implement it when a company starts growing. Humans, therefore, will overcorrect by default and that's why large corporations are (in more cases than not) irrationally-obstructive to getting work done.
+
+## What are the drawbacks?
+
+As a company with high autonomy and low coordination, shipping things that affect multiple teams is _a lot_ harder than shipping something within a single team. You may need extra meetings, or to form a temporary project team, or you may just fail!
+
+Since we handle a lot of data, at scale, this has meant performance has, well, um, been a performance to solve. Performance is a very broad area to solve for, ranging from our infrastructure all the way through to our UX, and thus involves a lot of coordination and dependencies.
+
+Before you start making changes, think hard about what your product should be optimized for.
+
+For example, if you are shipping the latest iPhone, complete design control is _more_ important to your users than having all the features. Likewise, if you're building small software product that has better UX than an incumbent, design and control matters.
+
+I _suspect_ you could still get higher quality with more autonomy – I'd probably have lots of designers and engineers collaborating with relatively few product managers, but I can't speak to the above as confidently since my experience is building a multi-product platform.
```

Comment:
```suggestion
For example, if you are shipping the latest iPhone, complete design control is _more_ important to your users than having all the features. 

Likewise, if you're building small software product that has better UX than an incumbent, design and control matters. I _suspect_ you could still get higher quality with more autonomy – I'd probably have lots of designers and engineers collaborating with relatively few product managers, but I can't speak to the above as confidently since my experience is building a multi-product platform.
```